src/scenarios_hpc_azure/launch_experiment.py

```diff
@@ -186,4 +209,19 @@ def launch():
         depend_on_task_ids=state_task_ids,
     )
     all_tasks_run += state_task_ids + postprocessing_tasks
-    launcher.azure_client.monitor_job(job_id, timeout=timeout_mins)
+    if dest_path is not None:
+        if os.path.exists(dest_path):
```

Comment:
hmmmm, if my `dest_path` is say `/azure_output`, the file will be stored at `/azure_output/{exp_name}/{job_id}` yes?

Reply:
correct